Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN apk add --no-cache openjdk8-jre php7 php7-json php7-openssl python py-lxml g
     && update-ca-certificates \
     && make install \
     && cp docker.config.php web/config.local.php \
+    && sed -i '/^upload_max_filesize/ s/=.*$/= 100M' \
```

Comment:
There seems to be a problem with the automated Docker build on docker hub: https://hub.docker.com/r/ubma/ocr-fileformat/builds/bvafgbah9zqysoblyvupmeg/

Reply:
Ping @kba 🔥

Reply:
Fixed. Since the image sizes don't differ now, I'll sumit my dev Dockerfile later.